scylladb/scylla-migrator
# docs: add sample site

## Motivation

Adds a [sample docs site](https://sphinx-theme.scylladb.com/) to the project.

## How to test

1. Make sure you meet the [prerequisites](https://sphinx-theme.scylladb.com/stable/getting-started/installation.html#prerequisites).
2. Clone the PR.
3. Go to the docs folder: `cd docs`
4. Build the docs: `make preview`
5. The site should render at http://127.0.0.1:5500/

## Next steps after merging

1. Add content to the site (@annastuchlik)
2. Request zendesk tag for this project, optional (@annastuchlik)
3. Configure the subdomain (@dgarcia360)
4. Configure GitHub Pages (@tarzanek @julienrf)

    Could you please enable the following option?

  > In the repository’s Settings, navigate to the Pages section and select GitHub Actions as the deployment source.
  > 
  > ![image](https://github.com/scylladb/scylla-migrator/assets/9107969/22073f3a-eaec-4e3f-9145-da99fd22de21)
  > Thanks!
